actions/image_partition_action.go

```diff
@@ -678,7 +679,17 @@ func (i *ImagePartitionAction) Verify(context *debos.DebosContext) error {
 		}
 	}
 
-	size, err := units.FromHumanSize(i.ImageSize)
+	// Calculate the size based on the unit (binary or decimal)
+	// binary units are multiples of 1024 - KiB, MiB, GiB, TiB, PiB
+	// decimal units are multiples of 1000 - KB, MB, GB, TB, PB
+	var getSizeValueFunc func(size string) (int64, error)
+	if regexp.MustCompile(`^[0-9]+[kKmMgGtTpP][iI][bB]+$`).MatchString(i.ImageSize){
+		getSizeValueFunc = units.RAMInBytes
```

Comment:
would it be worth renaming `RAMInBytes` to something more meaningful ?

Reply:
we can't do this as the the method name is defined in another repository https://github.com/docker/go-units/blob/master/size.go#L86